wls-ergebnismeldung-service/src/main/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/service/begruendung/BegruendungService.java

```diff
@@ -0,0 +1,61 @@
+package de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.begruendung;
+
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.Begruendung;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.BegruendungRepository;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.BezirkUndWahlIDStapelart;
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.util.ExceptionFactory;
+import jakarta.validation.constraints.NotNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.parameters.P;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class BegruendungService {
+
+    private final BegruendungRepository begruendungRepository;
+
+    private final BegruendungModelMapper begruendungModelMapper;
+
+    private final BegruendungValidator begruendungValidator;
+
+    private final ExceptionFactory exceptionFactory;
+
+    @PreAuthorize(
+        "hasAuthority('Ergebnismeldung_BUSINESSACTION_GetBegruendung')"
+                + " and @bezirkIdPermisionEvaluator.tokenUserBezirkIdMatches(#param.wahlbezirkID(), authentication)"
```

Comment:
```suggestion
                + " and @bezirkIdPermissionEvaluator.tokenUserBezirkIdMatches(#param.wahlbezirkID(), authentication)"
```
der typo wurde doch schon gefixed, dachte ich? bzw im [pr dazu](https://github.com/it-at-m/Wahllokalsystem/pull/531) steht was von einem folge-issue, was noch erstellt werden muss, gibt es das schon?

Reply:
korrekt, der typo wurde gefixt, das folgeissue ist erstellt #535, aber da sich der service sich die files der dependencies aus den releases baut, müssen wir erstmal auf das release warten. das wird zur folge haben, dass der service nicht mehr compiliert, sobald das release zu wls-common durch ist und genau dann müssen wir den typo hier fixen

Reply:
> das wird zur folge haben, dass der service nicht mehr compiliert, sobald das release zu wls-common durch ist und genau dann müssen wir den typo hier fixen

gibt es dazu auch schon ein issue, dass nach dem neuen wls-common release die services aktualisiert werden müssen? alternativ sollte in #535 noch als akzeptanzkriterium hinzugefügt werden, dass nach dem release die services angepasst wurden

Reply:
für den briefwahl service und wahlvorstand ja, für ergebnismeldung mach ich gleich eins

Reply:
issue #671